test/test_manager.py

```diff
@@ -39,6 +39,8 @@
 import libqtile.hook
 import libqtile.confreader
 
+import logging
```

Comment:
Haha, it should have been ordered alphabetically but still in the first block, as it is an import from the standard library. I should have mentioned it, sorry! But honestly that is no big deal, I'll take that on myself if you want, as we don't have any clear rule stated so far anyway.

Reply:
oh.. I'm sorry the simple my mistake. The rule is mentioned in pep8. Usually, Qtile developers are following that rule, I think.

Reply:
I fixed it!